src/Microsoft.Extensions.WebSockets.Internal/WebSocketConnection.cs

```diff
@@ -113,16 +113,14 @@ private static void Pinger(object state)
             {
                 // We don't need to wait for this task to complete, we're "tail calling" and
                 // we are in a Timer thread-pool thread.
-#pragma warning disable 4014
-                connection.SendCoreLockAcquiredAsync(
+                _ = connection.SendCoreLockAcquiredAsync(
```

Comment:
C# 7! Wildcard variable! Yaaaay! It's exactly like the `var ignore = [thing that returns a task]` @davidfowl has used in other places, just without the `var`, thanks to C# 7.